facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -92,12 +91,14 @@
     private static final String NO_CRASH_REPORT = "-noCrashReport";
     private static final String NO_SAVE_GAMES = "-noSaveGames";
     private static final String NO_SOUND = "-noSound";
+    private static final String SERVER_PORT = "-serverPort=";
 
     private static boolean isHeadless;
     private static boolean crashReportEnabled = true;
     private static boolean writeSaveGamesEnabled = true;
     private static boolean soundEnabled = true;
     private static boolean loadLastGame;
+    private static int serverPort = 25777;
```

Comment:
The default port is set in `TerasologyConstants.DEFAULT_PORT` - please use that one.